yb-voyager/cmd/exportData.go

```diff
@@ -208,12 +216,22 @@ func debeziumExportData(ctx context.Context, tableList []*sqlname.SourceName) er
 		snapshotMode = "initial"
 	}
 
-	var dbzmIncludeTableList []string
+	var dbzmTableList, dbzmColumnList []string
 	for _, table := range tableList {
-		dbzmIncludeTableList = append(dbzmIncludeTableList, table.Qualified.Unquoted)
+		dbzmTableList = append(dbzmTableList, table.Qualified.Unquoted)
 	}
 
-	utils.PrintAndLog("final table list for data export: %v\n", dbzmIncludeTableList)
+	utils.PrintAndLog("final table list for data export: %v\n", dbzmTableList)
+
+	for table, columns := range tablesColumnList {
+		for _, column := range columns {
+			if column == "*" {
+				dbzmColumnList = append(dbzmColumnList, source.Schema+"."+table+`[a-zA-Z0-9_."]+`)
```

Comment:
<schema>.<table>.* might also work, I think.

Reply:
Oh yes, that worked, actually, I tried this earlier but may be in a wrong way so that was not working earlier but this worked. Thanks!